Numbers.java

```diff
@@ -0,0 +1,79 @@
+package task1;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Scanner;
+
+public class Numbers {
+
+    ArrayList<Integer> listOfNumbers;
+
+
+    public Numbers(ArrayList<Integer> listOfNumbers) {
+        this.listOfNumbers = listOfNumbers;
+    }
+
+
+    public void addNumbers() {
+        Scanner reader = new Scanner(System.in);
+        System.out.println("Please enter number");
+        int number = reader.nextInt();
+        listOfNumbers.add(number);
+    }
+
+    public void whileLoopAddNumbers() {
+        Scanner reader = new Scanner(System.in);
+        char yOrN;
+        do {
+            addNumbers();
+            System.out.println("Do you wand to add another number? y/n");
+            yOrN = reader.next().charAt(0);
+        } while (yOrN == 'y');
+        System.out.print("Your list --> ");
+        forLoop(listOfNumbers);
+    }
+
+    private void forLoop(ArrayList<Integer> listOfNumbers) {
```

Comment:
Use other name for this method (without forLoop mention). Maybe: printAllNumbers